src/App.tsx

```diff
@@ -1,61 +1,132 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/userType';
+import { Todo } from './types/todoType';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUser(userId:number):User | null {
+  const foundUser = usersFromServer.find(user => userId === user.id);
+
+  return foundUser || null;
+}
+
+export const fullTodoInfo:Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(fullTodoInfo);
+  const [titleEntered, setTitleEntered] = useState(true);
+  const [userChosen, setUserChosen] = useState(true);
+
+  const generateTodoId = () => {
+    return Math.max(...visibleTodos.map(todo => todo.id)) + 1;
+  };
+
+  const handleReset = () => {
+    setUserId(0);
+    setTitle('');
+  };
+
+  const addTodo = () => {
+    const createNewTodo = {
```

Comment:
```suggestion
    const newTodo = {
```
Don't name it like a function